pages/dao/[symbol]/proposal/components/instructions/Raydium/RemoveLiquidityFromPool.tsx

```diff
@@ -76,38 +68,36 @@ const RemoveLiquidityFromPool = ({
           description: `${form.liquidityPool} LP Token Account could not be found for the selected Governance`,
         })
       }
-    })()
-  }, [form.governedAccount?.governance.pubkey, form.liquidityPool])
+    }
+    fetchLpMintInfo()
+  }, [governanceAccount?.governance.pubkey, form.liquidityPool])
 
   async function getInstruction(): Promise<UiInstruction> {
-    const isSerializable = await canSerializeInstruction({
-      form,
-      schema: removeRaydiumLiquidityPoolSchema,
-    })
+    const isSerializable = await canSerializeInstruction()
     if (!isSerializable || !form.liquidityPool || !lpMintInfo) {
```

Comment:
`form.liquidityPool || !lpMintInfo` can be checked for earlier, so that we don't unnecessarily run an async operation in these cases?